src/components/content/deployment/DeploymentInfomation.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { ProfileOutlined } from '@ant-design/icons';
+import { Descriptions } from 'antd';
+import React from 'react';
+import catalogStyles from '../../../styles/catalog.module.css';
+import oclDisplayStyles from '../../../styles/ocl-display.module.css';
+import { Deployment } from '../../../xpanse-api/generated';
+import { DeploymentScriptText } from '../common/ocl/DeploymentScript';
+import { DeploymentText } from '../common/ocl/DeploymentText';
+import { getDeployerToolIcon } from '../common/ocl/getDeployerToolIcon';
+
+function DeploymentInfomation({ deployment }: { deployment: Deployment }): React.JSX.Element {
+    return (
+        <>
```

Comment:
the gap on top must be reduced and kept uniform to other blocks. 

![Uploading image.png…]()